### PR TITLE
fix(ESSNTL-3378): make all apps consistent to handle insights-not-connected

### PR DESCRIPTION
**Pull request by mkholjuraev**
Resolves: https://issues.redhat.com/browse/ESSNTL-3378

To test:

1. Visit the inventory page
2. filter the table with 'insights-not-connected' from the Data-collector filter
3. Click on a system to open the details page
4. Observe that all tabs are consistent and display the following design.

image

### CODE REVIEW

**Comment by gkarat** [comment]
Didn't work for me: the message is not updated after navigating to any of the tabs.

[Screencast from 11-16-2022 02:12:29 PM.webm](https://user-images.githubusercontent.com/31385370/202189951-526d8ef4-6785-4019-a890-a5b216d33969.webm)

File: src/components/InventoryDetail/__snapshots__/AppInfo.test.js.snap [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`AppInfo should render correctly 1`] = `ReactWrapper {}`;
```
Comment (gkarat) [inline src/components/InventoryDetail/__snapshots__/AppInfo.test.js.snap:3]: I am afraid the snapshot is not correct: it's just an empty object. Did you mount the component the right way?

**Review by gkarat — CHANGES_REQUESTED** [review]

**Comment by mkholjuraev** [comment]
The UX behavior might change due. Putting this ticket on hold.

**Comment by mkholjuraev** [comment]
Closing this as the ticket is resolved on another duplicated one.

**Pull request closed without merging**